classes/class3/cat_homework/views/colorcats.handlebars

```diff
@@ -0,0 +1,8 @@
+<h2>All cats containing the color {{colorSearch}} from youngest to oldest.</h2>
+<ul>
+{{#each colorCats}}
+	{{#with this}}
+		<li>{{name}}, Age {{age}}, Colors: {{color}}</li>
+	{{/with}}
+{{/each}}
+</ul>
```

Comment:
Think about how you could have combined `cats.handlebars` and `colorcats.handlebars` into one template.